partiql-grammar/src/main/antlr/org/partiql/grammar/PartiQL.g4

```diff
@@ -119,15 +119,58 @@ removeCommand
 //  We essentially use the returning clause, because we currently support this with the SqlParser.
 //  See https://github.com/partiql/partiql-lang-kotlin/issues/708
 insertCommandReturning
-    : INSERT INTO pathSimple VALUE value=expr ( AT pos=expr )? onConflict? returningClause?;
+    : INSERT INTO pathSimple VALUE value=expr ( AT pos=expr )? onConflictClause? returningClause?;
 
 insertCommand
-    : INSERT INTO pathSimple VALUE value=expr ( AT pos=expr )? onConflict?  # InsertValue
-    | INSERT INTO pathSimple value=expr                                     # InsertSimple
+    : INSERT INTO pathSimple VALUE value=expr ( AT pos=expr )? onConflictClause?  # InsertLegacy
+    // See the Grammar at https://github.com/partiql/partiql-docs/blob/main/RFCs/0011-partiql-insert.md#2-proposed-grammar-and-semantics
+    | INSERT INTO symbolPrimitive asIdent? value=expr onConflictClause?           # Insert
```

Comment:
This doesn't completely match the proposed insert:
```antlr
<insert statement> ::= INSERT INTO <table name> [ AS <alias> ] 
    [  ( <attr name> [, <attr name> ]... ) ]
        <values>
    [ ON CONFLICT [ <conflict target> ] <conflict action> ]
```

Is the expectation to eventually modify `value` to be:
- DEFAULT VALUES
- VALUES <values>
- And specifically limiting the expr to be a SFW or bag literals?

Reply:
Good call out, we're expecting to have `expr` as `value` ultimately and evolve the grammar towards that; this decision was made to limit the scope of the RFC b/c there were just too many devils in details. Having said that, to keep the backward compatibility we're not regressing the AST at the moment; what I had in mind was to leave the AST as is and make limitation in planner instead which we can lift later on as we generalize the grammar. 

See, [the future possibilities of the RFC](https://github.com/partiql/partiql-docs/blob/main/RFCs/0011-partiql-insert.md#future-possibilities)

>In addition, we can extend the grammar, as explained in section 3.2 (Query), to any expression returning applicable PartiQL values; this will be more aligned with PartiQL/SQL++ generalization.